localdev/ches-mock/src/main.ts

```diff
@@ -0,0 +1,110 @@
+import sanitizeHtml from 'sanitize-html';
+import nodemailer from 'nodemailer';
+import express, { Request, Response } from 'express';
+
+const app = express();
+app.use(express.json());
+const port = 3025;
```

Comment:
let's get the port number from the environment variable but defaults to 3025 if not defined.